libraries/botbuilder-core/tests/testAdapter.test.js

```diff
@@ -498,17 +498,52 @@ describe(`TestAdapter`, function () {
     });
 
     it(`should return statuses from getTokenStatus`, function (done) {
-        const adapter = new TestAdapter((context) => {
-            context.adapter.getTokenStatus(context, 'user').then(statuses => {
+        const adapter = new TestAdapter(async (context) => {
+            try {
+                const statuses = await context.adapter.getTokenStatus(context, 'user');
                 assert(statuses);
                 assert(statuses.length == 2);
                 assert(statuses.reduce((j, status) => (j||status.ConnectionName === "ABC"), false));
+             
                 done();
-            });
+            } catch (err) {
+                done(err);
+            }
         });
 
         adapter.addUserToken('ABC', 'test', 'user', '123abc');
         adapter.addUserToken('DEF', 'test', 'user', 'def456');
         adapter.send('hi');
     });
+
+    it(`should throw when context parameter is not sent`, function (done) {
+        const adapter = new TestAdapter(async (context) => {
+            try {
+                await context.adapter.getTokenStatus();
+            } catch (err) {
+                assert(err.message === 'testAdapter.getTokenStatus(): context with activity is required',
+                `expected "testAdapter.getTokenStatus(): context with activity is required" Error message, not "${ err.message }"`);
+                done();
```

Comment:
I'm not a huge fan of this suggestion, but if the assert fails the test fails due to a timeout while waiting for `done` to be called.

You should be able to avoid this by doing the following:

```suggestion
                done(assert.strictEqual(err.message, 'testAdapter.getTokenStatus(): context with activity is required'));
```